apps/src/utils.js

```diff
@@ -387,3 +387,13 @@ exports.escapeText = function (text) {
 
   return returnValue;
 };
+
+/**
+ * Converts degrees into radians.
+ *
+ * @param degrees - The degrees to convert to radians
+ * @return `degrees` converted to radians
+ */
+exports.degreesToRadians = function (degrees) {
+    return degrees * (Math.PI / 180);
+};
```

Comment:
Ah shit, nice catch! Was originally returning the array, but when doc'ing the function decided an object would be better / more explicit. All of the images are larger than the bounding box so that line never got exercised. Oops!